src/usr/local/www/system_camanager.php

```diff
@@ -308,6 +317,7 @@
 
 		if (!$input_errors) {
 			write_config();
+			ca_trustedcerts_generate();
```

Comment:
See above, re; Agree to disagree then. My logic is sound and I've explained it thoroughly in several ways, consistently.

You can prefer your way, but that is not the way we want pfSense to behave.